tests/ValueParsers/IsoTimestampParserTest.php

```diff
@@ -181,13 +198,13 @@ public function validInputProvider() {
 			'-1-01-04T00:00:00Z' => array(
 				'-0000000000000001-01-04T00:00:00Z',
 				TimeValue::PRECISION_10a,
-				$gregorian,
+				$julian,
```

Comment:
I disagree. Magically turning dates into Julian for years where Julian did not even existed is surprising.